catalyst_voices/lib/pages/treasury/campaign_segment_controller.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:catalyst_voices/widgets/menu/voices_node_menu.dart';
+import 'package:flutter/material.dart';
+
+typedef ControllerBuilder = CampaignController Function(
```

Comment:
CampaignControllerBuilder? The reason is ControllerBuilder is quite generic and in flutter we have plenty of different controllers

Reply:
Agree